test-suites/test-aave/configurator.spec.ts

```diff
@@ -355,15 +408,82 @@ makeSuite('LendingPoolConfigurator', (testEnv: TestEnv) => {
     expect(liquidationThreshold).to.be.equal(strategyWETH.liquidationThreshold);
     expect(liquidationBonus).to.be.equal(strategyWETH.liquidationBonus);
     expect(stableBorrowRateEnabled).to.be.equal(strategyWETH.stableBorrowRateEnabled);
+    expect(borrowCap).to.be.equal(strategyWETH.borrowCap);
+    expect(supplyCap).to.be.equal(strategyWETH.supplyCap);
     expect(reserveFactor).to.be.equal(1000);
   });
 
-  it('Check the onlyLendingPoolManager on setReserveFactor', async () => {
+  it('Fails to change to too high borrowCap', async () => {
```

Comment:
Change title of the test - "Check that a borrow cap that exceeds the MAX_BORROW_CAP cannot be set"